.circleci/generate.sh

```diff
@@ -80,15 +81,18 @@ print_help()
 all=false
 free=false
 paid=false
+dedup=false
```

Comment:
all good but ... just semantics, shouldnt we rename this to "development" or something like that? I know it _deduplicates_, I just think it is better if we call it like what it is good for rather than what it does.

I think that all the naming can stay and we would just document "-d" flag as "development", up to you.

Reply:
Ah yes I like 'development' semantics better. Which ties to the comment above about mentioning it in the text +1